src/main/java/com/cloudbees/hudson/plugins/folder/AbstractFolder.java

```diff
@@ -497,6 +497,33 @@ public static <K, V extends TopLevelItem> Map<K, V> loadChildren(AbstractFolder<
         return configurations;
     }
 
+    @Override
```

Comment:
Note: actually required, or the `interface default` implementation will be used instead.